CHANGELOG.md

```diff
@@ -70,8 +70,9 @@ This version brings a breaking change with the move to use the [Zarr](https://za
 
 ## [0.7.1] - 2024-03-01
 
-### Fixed
+## Added
 
+- Added `AGAGE` as a source format in the `standardise_surface` function, and associated parser functions and tests. Reads in output files from Matt Rigby's agage-archive [git repository](https://github.com/mrghg/agage-archive).
```

Comment:
Can move this up in the CHANGELOG.md file now to align with the version - should go to somewhere around Line 9 (Below Unreleased above v0.8.2)